packages/eslint-plugin-odyssey/package.json

```diff
@@ -11,8 +11,11 @@
   "peerDependencies": {
     "eslint": ">=7.27.0 <=8"
   },
+  "dependencies": {
+    "known-css-properties": "^0.24.0"
+  },
```

Comment:
if we like this PR I'll just drop the dep and we'll version control the transformed properties in a separate utils module to keep things fast